### DISCUSSION TITLE
Guides on EPEL?

### DISCUSSION THREAD

**Discussion by tyteen4a03** [pypa/manylinux#1409] [General]
Are there any guides on how to get EPEL working on manylinux builds?

**Comment by mayeut** [comment]
You can follow guides on how to setup EPEL repo for the various base images mentioned in the readme (e.g. for manylinux2014, follow the [guide for centOS 7](https://www.google.com/search?q=centos+7+epel)).

**Reply by tyteen4a03** [reply]
Do you have guides for Musllinux? Docs on this seems to be nonexistent.

**Reply by mayeut** [reply]
[EPEL](https://docs.fedoraproject.org/en-US/epel/) exists for RedHat based distros. I don't know of something similar for alpine based distros (but I'm not using those personally so might be missing something)
